admin/src/scenes/centers/view/Nav.js

```diff
@@ -38,16 +36,18 @@ export default function Nav({ center, tab, onChangeCohort, onChangeTab, focusedS
             style={{ borderRadius: "0.5rem 0 0 0.5rem" }}>
             Équipe
           </Tab>
-          <Tab
-            isActive={tab === "volontaires"}
-            middle
-            onClick={() => {
-              onChangeTab("volontaires");
-              history.push(`/centre/${center._id}/volontaires`);
-            }}
-            style={{ borderLeft: "1px solid rgba(0,0,0,0.1)", borderRight: "1px solid rgba(0,0,0,0.1)", minWidth: "110px" }}>
-            Volontaires
-          </Tab>
+          {user.role !== "head_center" ? (
```

Comment:
apres reflexion, je pense quil vaut mieux partir sur une white list qu'une black list. qu'en penses-tu ?
on a eu plusieurs fois le scénario ou on oublie que ya ce sujet la et ducoup un role a accès a des infos
```suggestion
          {[ROLES.ADMIN, ROLES.REFERENT_REGION, ROLES.REFERENT_DEPARTMENT].includes(user.role) ? (
```